docker-compose.yml.dist

```diff
@@ -12,4 +12,4 @@ services:
       - ./tools:/lfs/tools
       - ./lfs:/lfs
       - ./dist:/dist
-    entrypoint: ["sleep", "inf"]
+    entrypoint: ["bash", "/book/book.sh"]
```

Comment:
> Good, now need to find why it's not compiled and why ISO is empty :)

I will leave that upto you. Let me know when this is done so that I can complete the release script.